mne/channels/montage.py

```diff
@@ -406,31 +512,46 @@ class DigMontage(object):
     hsp : array, shape (n_points, 3)
         The positions of the headshape points in 3d.
         These points are in the native digitizer space.
+        Deprecated, will be removed in 0.20.
     hpi : array, shape (n_hpi, 3)
         The positions of the head-position indicator coils in 3d.
         These points are in the MEG device space.
+        Deprecated, will be removed in 0.20.
     elp : array, shape (n_hpi, 3)
         The positions of the head-position indicator coils in 3d.
         This is typically in the native digitizer space.
+        Deprecated, will be removed in 0.20.
     point_names : list, shape (n_elp)
         The names of the digitized points for hpi and elp.
+        Deprecated, will be removed in 0.20.
     nasion : array, shape (1, 3)
-        The position of the nasion fidicual point.
+        The position of the nasion fiducial point.
+        Deprecated, will be removed in 0.20.
     lpa : array, shape (1, 3)
-        The position of the left periauricular fidicual point.
+        The position of the left periauricular fiducial point.
+        Deprecated, will be removed in 0.20.
     rpa : array, shape (1, 3)
-        The position of the right periauricular fidicual point.
+        The position of the right periauricular fiducial point.
+        Deprecated, will be removed in 0.20.
```

Comment:
Such a brutal deprecation of all params really suggests we should just use a different class entirely (e.g., a new one, like perhaps `Digitization`). :(